tests/autologging/fixtures.py

```diff
@@ -1,8 +1,26 @@
 import os
+import sys
 
 import pytest
 
-from mlflow.utils.autologging_utils import _is_testing, _AUTOLOGGING_TEST_MODE_ENV_VAR
+import mlflow.utils.logging_utils as logging_utils
+from mlflow.utils.autologging import _is_testing, _AUTOLOGGING_TEST_MODE_ENV_VAR
+
+
+PATCH_DESTINATION_FN_DEFAULT_RESULT = "original_result"
+
+
+@pytest.fixture
+def patch_destination():
+    class PatchObj:
+        def __init__(self):
+            self.fn_call_count = 0
+
+        def fn(self, *args, **kwargs):
+            self.fn_call_count += 1
+            return PATCH_DESTINATION_FN_DEFAULT_RESULT
+
+    return PatchObj()
```

Comment:
Moved this out of `tests/autologging/test_autologging_safety_unit.py` for reusability across test modules.